src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/DSLMethodStrategy.java

```diff
@@ -34,7 +36,11 @@ public String toDSL() {
             if (isParentAMethod) {
                 return getStrategyForObject(propertyDescriptor).toDSL();
             }
+            if (propertyDescriptor.getParent().getName().equals("hudson.model.StringParameterDefinition") && propertyDescriptor.getName().equals("defaultValue")) {
```

Comment:
In the dynamic dsl, the properties under string param are methods and in the older syntax they are parameters, so I added a check in both method and parameter strategy.

Reply:
hm yeah like i said in our meeting i don't really like doing this in the base method strategy file. It's way too specific and deals with 1 type of attribute. I'd rather have a custom class for force-into-string params and methods